src/server/services/metadata.ts

```diff
@@ -247,6 +247,7 @@ export const countriesList = [
   { text: "Slovenia", value: "Slovenia", code: "SVN" },
   { text: "Solomon Islands", value: "Solomon Islands", code: "SLB" },
   { text: "Somalia", value: "Somalia", code: "SOM" },
+  { text: "Somaliland", value: "Somaliland", code: "SOM" },
```

Comment:
will this have any adverse effects? I forget if we're translating from a code into text/value. In which case there may be some confusing between SOM for Somalia and SOM for Somaliland

Reply:
So I did some research and wasn't able to find a iso3 country code for Somaliland. I don't know if it has one but we're using the code for AWS location stuff.

So potentially if someone selects Somaliland and puts in a Somalia region they'll get Somalia results.

Reply:
FYI, northern Cyprus is set to CYP (same as Cyprus) so it should be the same here.  On that point @RichardBray , it looks like northern Cyprus is now spelt with an uppercase N in the countriesList.  It should be lowercase.  Can you check that.

Reply:
Should North Macedonia have a lowercase n as well?

Reply:
Nope.  That's fine as it is.